### ISSUE TITLE
Give better guidance on content design

### ISSUE THREAD

**Issue by mgifford** [mgifford/ACCESSIBILITY.md#36]
There is a lot of good guidance on content design, it applies to accesibility too as a well constructed page content will be easier to understand. ACCESSIBILITY.md doesn't have any guidance on that, beyond plain language.

Some great government ones:

- https://hub.innovation.ca.gov/content-design/odi-style-guide/index.html
- https://design.canada.ca/style-guide/
- https://hub.innovation.ca.gov/content-design/recommended-reading/index.html
- https://www.gov.uk/guidance/style-guide
- https://design.homeoffice.gov.uk/design-and-content/content/content-style-guide
- https://service-manual.ons.gov.uk/content/writing-for-users/structuring-content

These look useful too:

- https://blog.hubspot.com/blog/tabid/6307/bid/30557/6-guidelines-for-exceptional-website-design-and-usability.aspx
- https://www.uxdesigninstitute.com/blog/content-design-principles/
- https://contentsquare.com/guides/web-design/best-practices/
- https://postiz.com/blog/content-creation-best-practices
- https://uxwritinghub.com/content-style-guides/
- https://www.wearetg.com/blog/website-content/

I want a page with some high level guidance on how to create content that is easy to consume and aligned with our current accessibility goals.
